client/layout/masterbar/logged-in.jsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Masterbar from './masterbar';
+import Item from './item';
+import StatsItem from './stats-item';
+import Publish from './publish';
+import Notifications from './notifications';
+import Gravatar from 'components/gravatar';
+import layoutFocus from 'lib/layout-focus';
+import config from 'config';
+
+export default React.createClass( {
+	displayName: 'Masterbar',
+
+	propTypes: {
+		user: React.PropTypes.object,
+		sites: React.PropTypes.object,
+		section: React.PropTypes.oneOfType( [ React.PropTypes.string, React.PropTypes.bool ] ),
+	},
+
+	getInitialState() {
+		return {
+			// whether we show the notifications panel
+			showNotifications: false,
+		};
+	},
+
+	clickMySites() {
+		layoutFocus.setNext( 'sidebar' );
+	},
+
+	clickReader() {
+		layoutFocus.setNext( 'content' );
+	},
+
+	clickNotifications() {
+		this.setState( {
+			showNotifications: ! this.state.showNotifications
+		} );
+	},
+
+	isActive( section ) {
+		return section === this.props.section && ! this.state.showNotifications;
+	},
+
+	wordpressIcon() {
```

Comment:
nitPicky :-)

I'll respectfully stick with the lower-case p here. We shouldn't do `WordPressIcon()` for a method name, and IMO `wordPressIcon()` isn't any better than `wordpressIcon()`. So let's stick with the latter and succumb to `snakeCase`.

`</BikeShed>`